CleverTapSDK/CTLocalDataStore.m

```diff
@@ -610,6 +602,54 @@ - (void)removeProfileFieldsWithKeys:(NSArray *)keys {
     [self removeProfileFieldsWithKeys:keys fromUpstream:NO];
 }
 
+- (BOOL)isEventLoggedFirstTime:(NSString*)eventName {
+    @synchronized (self.userEventLogs) {
+    // TODO: add normalisation 
+        if ([self.userEventLogs containsObject:eventName]) {
+            return NO;
+        }
+    }
+    // TODO: Add normalized name here
+    NSInteger count = [self.dbHelper getEventCount:[CTUtils getNormalizedName:eventName] deviceID:self.deviceInfo.deviceId];
+    if (count > 1) {
+        @synchronized (self.userEventLogs) {
+            [self.userEventLogs addObject:eventName];
```

Comment:
Add name normalized into set

Reply:
@nzagorchev  I was in two minds about this too. Android SDK adds it without normalizing as well too.

https://github.com/CleverTap/clevertap-android-sdk/blob/0727511d7d2acbcef085478ef16078146fd0c49f/clevertap-core/src/main/java/com/clevertap/android/sdk/LocalDataStore.java#L285